css/style.css

```diff
@@ -25,11 +25,15 @@ body {
   width: 60px;
```

Comment:
You're moved onto stage 2 without completing stage 1.  Stage one requires that you make your calculator look as close to the image provided as possible. 

- You don't have the right text or background colors
- Your borders are too think
- more…

Please go back to stage 1 and do your best to make a pixel perfect replica.